README.md

```diff
@@ -95,20 +98,175 @@ await org.save()
 await org.delete()
 
 
-# Fetch org
+# Get org
 org3 = await Org.get(org2.id)
 assert org3 == org2
 
-# Fetch a list of orgs
+# Get a list of orgs
 orgs = await Org.get_list()
 
-# Fetch a filtered list of orgs
+# Get a filtered list of orgs
 orgs = await Org.get_list(tag="green")
 
-#  Fetch orgs that have several different tags
+# Get orgs that have several different tags
 orgs = await Org.get_list(tag=["green", "blue", "red"])
+
+# Fetch the latest state of a table in the db
+org3.name = "fake name"
+await org3.fetch()
+assert org3.name == "new org2"
+
+```
+
+#### Serialization
+
+DatabaseModels have their own serialization functions to convert to and from
+dictionaries.
+
+```python
+# Serializing org with `to_dict()`
+org = Org.create(name="org serialize", slug="org-serialize")
+org_dict = org.to_dict()
+expected_org_dict = {
+    "id": "e64f6c7a-1bd1-4169-b482-189bd3598079",
+    "name": "org serialize",
+    "slug": "org-serialize",
+    "created_at": 2018-01-01 7:03:45
+    "updated_at": 2018-01-01 9:24:12
```

Comment:
update syntax here

Reply:
for the timestamp? What should it be

Reply:
Idk, a string or something. It's not valid Python as-is